Process.h

```diff
@@ -113,7 +115,7 @@ typedef struct ProcessFieldData_ {
    const char* name;
    const char* title;
    const char* description;
-   int flags;
+   uint32_t flags;
```

Comment:
Please also change the type at
https://github.com/htop-dev/htop/blob/46a2e8ac63fb758a589a82d4dca60de1aa0c1eb2/Settings.c#L93
https://github.com/htop-dev/htop/blob/46a2e8ac63fb758a589a82d4dca60de1aa0c1eb2/Settings.h#L30